### PR TITLE
Gflags 2.2 premake

### PR DESCRIPTION
**Pull request by msbit**
To deal with https://github.com/google/guetzli/issues/61.

Looking over the MacPorts Portfile (https://github.com/macports/macports-ports/blob/master/devel/gflags/Portfile) it looks like there is no .pc file generated as part of the process, as this was only packed in since gflags 2.2.

This will cause issues for gflags 2.0/2.1 users on Debian/Ubuntu as the package maintainers generate the libgflags.pc file as part of their packaging.

Unsure about how other distributions package gflags.

Open to any input on how to make this more portable.

____

Signed the Google Individual CLA on Mar 17, 2017 18:47 PDT

### CODE REVIEW

**Comment by msbit** [comment]
> Signed the Google Individual CLA on Mar 17, 2017 18:47 PDT

File: guetzli.make [hunk 1]
```diff
@@ -11,20 +11,28 @@ endif
 .PHONY: clean prebuild prelink
 
 ifeq ($(config),release)
-  RESCOMP = windres
+  ifeq ($(origin CC), default)
```
Comment (robryk) [inline guetzli.make]: Please regenerate with --os=linux. That generates makefiles that appear to wotk on Mac, but --os=macosx generates makefiles that don't work on a typical Linux.

**Comment by robryk** [comment]
Thank you!

**Comment by msbit** [comment]
@robryk re-ran premake5 with `cc=gcc` as I was testing under linux just then, and rebase squashed. Generates the same output as `os=linux` from what I can tell.

**Pull request merged**